addons/web/static/src/views/fields/radio/radio_selection_field_with_filter.js

```diff
@@ -23,7 +23,7 @@ export const radioSelectionFieldWithFilter = {
     extractProps({ options }, { context: { allowed_selection } }) {
         return {
             ...radioField.extractProps(...arguments),
-            allowed_selection: allowed_selection,
+            allowed_selection: new Set(allowed_selection),
```

Comment:
Wouldn't it be better to make the python return a set too then?

Reply:
Over jsonrpc the python set it going to be converted to a JSON array anyway. I agree it is not super consistent python vs js but I was annoyed by the O(n²) inside `items` even if I guess `n` will always be rather low. What do you think?

Reply:
Actually, when is `extractProps` called? Do we pay this `new Set(...)`'s O(n) every time the widget is rendered?

Reply:
Oh well, let's just keep an Array